tests/clusters/test_cluster.py

```diff
@@ -23,7 +23,7 @@
 
 ALL_STORAGE_SETS = {
     "outcomes",
-    "transactions_ro",
+    "transaction_ro",
```

Comment:
Why not just delete this line?

Reply:
I will, I was just making sure that having non-existent storage set keys in cluster settings wasn't going to break anything